corehq/apps/domain/models.py

```diff
@@ -1044,3 +1044,17 @@ def update_calculations(cls, domain, property_to_update):
         # update_fields prevents the possibility of a race condition
         # https://stackoverflow.com/a/1599090
         obj.save(update_fields=[property_to_update])
+
+
+class ProjectLimitType():
+    LIVE_GOOGLE_SHEETS = 'lgs'
+
+    CHOICES = (
+        (LIVE_GOOGLE_SHEETS, "Live Google Sheets"),
+    )
+
+
+class ProjectLimit(models.Model):
+    domain = models.CharField(max_length=256, db_index=True, unique=True)
```

Comment:
I realized we probably don't want this to be `unique=True` because there can be multiple `limit_type`s for a `domain`